Gemfile

```diff
@@ -19,6 +19,7 @@ gem 'bootsnap'
 group :development, :test do
   gem 'pry'
   gem 'figaro'
+  gem 'faraday'
```

Comment:
This is fine for the practice assessment, but usually Faraday will need to go outside of the `:development, :test` group